t/i18n.t

```diff
@@ -14,30 +14,45 @@ my class I18NAwareForm is Cro::WebApp::Form {
     has $.name is rw is i18n-label('name-field');
 }
 
-my $application = route {
-    load-translation-file('main', 't/resources/test.po');
+is get-response(route {
+    load-translation-file('main', 't/resources/main.po');
     _-prefix 'main';
+
     get -> 'render' {
         is 'b', _('a', :prefix('main'));
         template 'i18n-_.crotmp';
     }
+}), "b\nb";
+
+ok get-response(route {
+    load-translation-file('main', 't/resources/main.po');
+    _-prefix 'main';
 
-    get -> 'form' {
+    get -> 'render' {
         template 'i18n-form.crotmp', { foo => I18NAwareForm.empty };
     }
-}
-my $server = Cro::HTTP::Server.new(:$application, :host('localhost'), :port(TEST_PORT));
-$server.start;
-LEAVE try $server.stop;
-my $client = Cro::HTTP::Client.new(base-uri => "http://localhost:{ TEST_PORT }", :cookie-jar);
-
-my $render-response;
-lives-ok { $render-response = await $client.get("/render") },
-        'Can use _ in a template';
-ok $render-response.defined;
-is await($render-response.body-text), "b\nb";
-
-lives-ok { $render-response = await $client.get("/form") },
-        'Can render a form in a template';
-ok $render-response.defined;
-ok await($render-response.body-text) ~~ /'Your Name'/;
\ No newline at end of file
+}) ~~ /'Your Name'/;
+
+is get-response(route {
+    # XXX We currently fuzzy-match `en` and `en-XX`, should we really?
+    load-translation-file('main', 't/resources/main.po', :language<en en-GB en-US>);
+    load-translation-file('main', 't/resources/main-fr.po', :language<fr fr-FR fr-CH>);
+    _-prefix 'main';
```

Comment:
Hm, this looks delightfully odd. I'm tempted to suggest `translation-prefix`, but then it's maybe less clear that it's about the `_` function...perhaps it's fine.

Reply:
Definitely To Be Nitpicked™.